CHANGELOG.md

```diff
@@ -1,7 +1,7 @@
 ## main
 
 ### Features
-
```

Comment:
prettier is unhappy 😀 

Also, please sort it alphabetically

Reply:
done